sh_scrapy/settings.py

```diff
@@ -222,5 +223,12 @@ def _load_default_settings(settings):
     }, priority='cmdline')
 
 
+def _enforce_required_settings(settings):
```

Comment:
@chekunkov If you're fine with the function name and added tests, let's merge it, I have nothing to add.